src/Cryptol/Parser/Position.hs

```diff
@@ -22,7 +22,10 @@ import Control.DeepSeq
 import Cryptol.Utils.PP
 
 data Located a  = Located { srcRange :: !Range, thing :: !a }
-                  deriving (Eq, Show, Generic, NFData)
+                  deriving (Eq, Generic, NFData)
+
```

Comment:
I can make the Show instance into an instance of ShowAST, and it works. I've restored the Show derivation for Located.